packages/-ember-data/addon/store.ts

```diff
@@ -1 +1,21 @@
-export { default } from '@ember-data/store';
+import Store from '@ember-data/store';
+import { RecordData } from '@ember-data/record-data/-private';
+import { identifierCacheFor } from '@ember-data/store/-private';
+import { IDENTIFIERS } from '@ember-data/canary-features';
+
+type RecordDataStoreWrapper = import('@ember-data/store/-private/ts-interfaces/record-data-store-wrapper').RecordDataStoreWrapper;
```

Comment:
Non-blocking: can we import from `@ember-data/store/-private` instead?

Reply:
we would have to incur the bloat of re-exporting our interfaces there, so this seems better ?